dockerfiles/onnxruntime-tensorrt.dockerfile

```diff
@@ -32,14 +48,19 @@ RUN git clone --single-branch --branch ${ONNXRUNTIME_BRANCH} --recursive ${ONNXR
     rm -rf onnxruntime cmake-3.14.3-Linux-x86_64
 
 # opencv deps
-RUN apt install -y libsm6 libxext6 libxrender-dev ffmpeg x264 libx264-dev libsm6
+RUN apt update && apt install -y libsm6 libxext6 libxrender-dev ffmpeg x264 libx264-dev libsm6
 
 WORKDIR /app/
 
 # install vortex-runtime
-
 COPY src/runtime src/runtime
 COPY examples examples
 # note dont provide extras since onnxruntime tensorrt need to be installed from source
 # need to manually install onnx instead of using extras
-RUN cd src/runtime && pip install onnx==1.6.0 && pip install -e .
\ No newline at end of file
+RUN cd src/runtime && pip install -U setuptools onnx==1.6.0 && pip install -e .
+
+FROM runtime AS development
+# install vortex-development
+COPY src/development src/development
+RUN pip install --upgrade pip
+RUN cd src/development && pip install --ignore-installed --timeout=10000 -e .
```

Comment:
could you please add testing to check that votex is installed and tensorrt runtime is available?

Reply:
moving tensor to cpu actually performs synchronization https://pytorch.org/docs/stable/notes/cuda.html#asynchronous-execution

> PyTorch automatically performs necessary synchronization when copying data between CPU and GPU or between two GPUs

so i think it is better let pytorch decide the synchronization

Reply:
okay

Reply:
updated